x-pack/plugins/cross_cluster_replication/public/app/components/follower_index_form/follower_index_form.js

```diff
@@ -579,6 +582,7 @@ export const FollowerIndexForm = injectI18n(
                       type={type}
                       areErrorsVisible={areErrorsVisible}
                       onValueUpdate={this.onFieldsChange}
+                      dataTestSubj={`ccrFollowerIndexForm${field.charAt(0).toUpperCase() + field.slice(1)}Input`}
```

Comment:
Also, can we provide a hardcoded test-subject value via the advanced setting config? This will make tests which depend upon these selectors more robust in the event that we change the `field` value at some point (and it will have the added benefit of making the code more explicit and thereby easier to read).

```js
const {
  field,
  title,
  description,
  label,
  helpText,
  defaultValue,
  type,
  testSubject, // we'll have to add this to each advanced setting config object
} = advancedSetting;
```